src/pages/publication/PublicationList.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { FC } from 'react';
+
+import styled from 'styled-components';
+import { DummyPublicationListElement } from './MyPublications';
+import { Table, TableBody, TableCell, TableHead, TableRow } from '@material-ui/core';
+import { useTranslation } from 'react-i18next';
+
+interface StyledTableRowProps {
+  index: number;
+}
+const StyledTableRow = styled(TableRow)<StyledTableRowProps>`
+  background-color: ${props =>
+    Boolean(props.index % 2) ? props.theme.palette.box.main : props.theme.palette.background.default};
```

Comment:
liker denne bedre enn !!. Kjør debatt ...

Reply:
Funker for meg.
Men vil nevne at vi kunne gjort dette med plain CSS ogs, ved å lage en parent med `nth-child(2) { ... }`. Da hadde vi sluppet props as well :)

Reply:
den er bedre, ja